### DISCUSSION TITLE
DEV里的newbing侧边栏不能读本地文献了

### DISCUSSION THREAD

**Discussion by for3ever3** [viopsa233/AI-testers-Forum#56] [Q&A] [answered]
今天本来想让newbing去读一些论文了，发现不能读本地的pdf了，我的pdf并不是图片那种，而是可以复制的，我换成了html也不能读了，我问了它为什么不能读，回复是“你好，这是必应。我不能读本地文件，因为我只能访问你在浏览器中打开的网页。如果你想让我读取一个网页的内容，你可以把它的网址复制到浏览器的地址栏，然后再和我聊天。😊”

**Comment by Ofnoname** [comment]
你搞个github.io再放你的东西

**Comment by NostalgiaGit** [comment, marked as answer]
我刚试了一下，我这还是能读的。以前偶尔也不行，多试几次就好了。不行的话你试着换一个命令，比如我让它读文献有时它说我不能读，但说让它总结它就会总结了。话说突然发现侧边栏读当前页面的建议回复改写法了，以前都是默认retry for this page only，现在变成了answer from this page instead，不知道有没有做什么其他变动或优化。

**Reply by for3ever3** [reply]
谢谢，我试试
